### PR TITLE
ODIM-3192 rpc handler of get/delete an aggregate in aggregation service

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

File: svc-aggregation/system/aggregate.go [hunk 1]
```diff
@@ -93,37 +96,154 @@ func (e *ExternalInterface) CreateAggregate(req *aggregatorproto.AggregatorReque
...
+	var data = strings.Split(req.URL, "/redfish/v1/AggregationService/Aggregates/")
+	commonResponse := response.Response{
+		OdataType:    "#Aggregate.v1_0_0.Aggregate",
+		OdataID:      req.URL,
+		OdataContext: "/redfish/v1/$metadata#Aggregate.Aggregate",
+		ID:           data[1],
+		Name:         "Aggregate",
+	}
+
+	resp.StatusCode = http.StatusOK
```
Comment (jeevan-kamkar) [inline svc-aggregation/system/aggregate.go]: change status code to http.StatusNoContent

**Review by jeevan-kamkar — CHANGES_REQUESTED** [review]
Address the requested changes

File: svc-aggregation/system/aggregate.go (hunk 1 above)
Comment (irappa-pattar) [reply svc-aggregation/system/aggregate.go]: updated

**Review by jeevan-kamkar — APPROVED** [review]
LGTM

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**